xarray/convert.py

```diff
@@ -49,3 +62,108 @@ def set_cdms2_attrs(var, attrs):
     cdms2_var = cdms2.createVariable(var.values, axes=axes, id=dataarray.name)
     set_cdms2_attrs(cdms2_var, var.attrs)
     return cdms2_var
+
+
+# TODO: Add converting bounds from xarray to Iris and back
+def to_iris(dataarray):
+    """Convert a DataArray into a Iris Cube
+    """
+    # Iris not a hard dependency
+    import iris
+    import iris.fileformats._pyke_rules.compiled_krb.fc_rules_cf_fc \
```

Comment:
Can we add some sort of guard so this doesn't fail if/when Iris cleans up it's internals and moves this function? We've been bitten by using private APIs before.

Ideally, @pelson would add something exposing this to the public API for the next release, and we could use something like:
```python
try:
    from iris.fileformats import parse_cell_methods
except ImportError:
    # prior to Iris 2.0 (or whatever)
    from iris.fileformats._pyke_rules.compile_krb.fc_rules_cf_fc \
        import _parse_cell_methods as parse_cell_methods
```